### PR TITLE
HW01

### PR DESCRIPTION
**Pull request by Luorao**
I think this hw guides us to go through all the procedures, it is a fair one.

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* Commit regularly, like you did, is a really great practice!
* You completed all the HW requirements. Great job :)
* Just for a future reference, the comment on how you changed the repo suppose to be on the comment section when you pull request.

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* Commit regularly, like you did, is a really great practice!
* You completed all the HW requirements. Great job :)
* Just for a future reference, the comment on how you changed the repo suppose to be on the comment section when you pull request.

**Review by Yanchi-Liu — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |          X |              |            |
| **Coding strategy**         |          X |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |          X |              |            |
| **Ease of access**          |   X        |              |            |

Very good job!
You have accomplished all the tasks required with a high quality. 
Hope every assignment of your future well!

File: HW-01.Rmd [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "HW-01"
```
Comment (wanitchayap) [inline HW-01.Rmd:2]: You actually should not use Rmd for this HW since we are not writing any code

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+I am a second-year MACRM student from Harris School. My goal is to improve my coding skills. I mainly worked in **STATA** in the past and had done some **R**. (By the way, most people I know write code when they do **STATA**. So I was pretty surprised to see the *GUI* side of it.)
+
+My research interests are in applied Economics so skills in data science become a must for me. I will use this course to:
+* Advance my coding skills in **R**
+* Possibly get to know some machine learning technique? 
+* Meet people with similar interests!
+
+***
+## How did I change repo?
+1. pull from the homework repo, and clone it using *SSH* link, you can find it on the right green button on  [hw01](https://github.com/Luorao/hw01). I use this instead of the *https* after setting up the SSH Key.
```
Comment (wanitchayap) [inline README.md:13]: Great link embedding :)

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+## Bio
+
+I am a second-year MACRM student from Harris School. My goal is to improve my coding skills. I mainly worked in **STATA** in the past and had done some **R**. (By the way, most people I know write code when they do **STATA**. So I was pretty surprised to see the *GUI* side of it.)
+
+My research interests are in applied Economics so skills in data science become a must for me. I will use this course to:
+* Advance my coding skills in **R**
+* Possibly get to know some machine learning technique? 
+* Meet people with similar interests!
+
+***
```
Comment (wanitchayap) [inline README.md:11]: Love the section separation here

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+
+## Bio
+
+I am a second-year MACRM student from Harris School. My goal is to improve my coding skills. I mainly worked in **STATA** in the past and had done some **R**. (By the way, most people I know write code when they do **STATA**. So I was pretty surprised to see the *GUI* side of it.)
```
Comment (Yanchi-Liu) [inline README.md:4]: A tiny suggestion that the bio will be better if you include your name in it!

**Pull request closed without merging**